README.md

```diff
@@ -0,0 +1,40 @@
+# triSYCL
+
+triSYCL is an implementation of the Khronos [SYCL 2.2](https://www.khronos.org/registry/SYCL/specs/sycl-2.2.pdf) standard.
+
+## What is SYCL?
+
+SYCL is an open standard for running C++11 programs on heterogeneous computing devices, including:
+- GPUs
+- distributed CPUs
+- embedded devices
+- FPGAs
+- (your cool new device here :-) )
+
+## Example
+
+```
+auto cg = [&]( handler & h) {
+    auto accA = buf . get_access < access::mode::read >( h) ;
+    auto accB = buf . get_access < access::mode::write >( h );
+    h. parallel_for < class myKernel >( myRange , [=]( item it ) {
+        accA [ it ] = accB [ it ];
+    }) ;
+};
+someQueue.submit ( cg ) ;
```

Comment:
You love spaces. :-)
I am not sure using a separate `cg` variable adds a lot here.
Could be just:
```
queue {}.submit([&] handler h) {
/* ... */
```

Reply:
Changed with https://github.com/keryell/triSYCL/commit/b18676a559d46775ae212574dd46d55ac5981ed4